ndcube/ndcube.py

```diff
@@ -489,12 +446,24 @@ def crop_by_coords(self, lower_corner, interval_widths=None, upper_corner=None,
               for i in range(self.data.ndim)])
         all_world_corners = [all_world_corners_grid[i].flatten()*lower_corner[i].unit
                              for i in range(n_dim)]
+
+        # Convert them back to units of world_axis_units
+        world_axis_units = self.high_level_wcs.low_level_wcs.world_axis_units[::-1]
```

Comment:
We are asserting that `self.wcs` is a low level wcs object right?
```suggestion
        world_axis_units = self.wcs.world_axis_units[::-1]
```